apps/judicial-system/web/src/components/Table/SortButton/SortButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import cn from 'classnames'
+
+import { Box, Icon, Text } from '@island.is/island-ui/core'
+
+import * as styles from '../Table.css'
+
+interface Props {
+  title: string
+  onClick: () => void
+  sortAsc: boolean
+  sortDes: boolean
+}
+
+const SortButton: React.FC<Props> = (props) => {
+  const { title, onClick, sortAsc, sortDes } = props
+
+  return (
+    <Box
+      component="button"
+      display="flex"
+      alignItems="center"
+      className={styles.thButton}
+      onClick={onClick}
+    >
+      <Text fontWeight="regular">{title}</Text>
+      <Box
+        className={cn(styles.sortIcon, {
+          [styles.sortAccusedNameAsc]: sortAsc,
```

Comment:
Nice! Consider renaming the styles for the sortIcon. Since you've extracted the component there's no longer a need for having separate style names for each one, so they can now be .sortAsc and sortDes